intake/conftest.py

```diff
@@ -123,24 +125,24 @@ def http_server():
         p.communicate()
 
 
-@pytest.fixture()
+@pytest.fixture(scope='function')
```

Comment:
I thought that was the default...

Reply:
Indeed. I suspect I was trying to track something down and couldn't remember so was being explicit. I'll undo that

Reply:
It is fine to leave it, the Zen says that explicit is always better.